### ISSUE TITLE
How do we connect minted NFTs from the CLI to a Metaplex store to be auctioned off?

### ISSUE THREAD

**Issue by numisman** [metaplex-foundation/metaplex#362]
Hey folks - I am attempting to autogenerate 10K NFTs and sell them in a metaplex store.

I am able to mint and see the NFTs in my devnet wallet, but when I spin up a metaplex store with that wallet as the REACT_APP_STORE_OWNER_ADDRESS_ADDRESS variable value in .env I do not see the minted tokens - even though I've confirmed I'm using the correct wallet with Phantom.

So, am I going about this all wrong?

What is the link that attaches a Metaplex store to the minted tokens you minted through the candy store CLI?

**Comment by numisman** [comment]
![image](https://user-images.githubusercontent.com/87335441/132916865-065ed8eb-533f-450a-97b4-d3445b179d01.png)
This is the candy machine I used: 
        "uuid": "5TLbRP",
        "config": "5TLbRPMp13fuVqDQ2ZX7E28mENQabfY6eEWzHW97UMgw"
So how do I connect it up?

**Comment by TreantBG** [comment]
Yeah im stuck here as well. Maybe is not developed yet

**Comment by stegaBOB** [comment]
I have the same issue, specifically with the candy machine

**Comment by stegaBOB** [comment]
wow. Massive brainfart here. Since we didn't mint the NFTs ourselves, the candy machine has to be added as one of the creators in order for it to show up. I'm guessing that was all of our issues.

**Comment by lz2470** [comment]
> wow. Massive brainfart here. Since we didn't mint the NFTs ourselves, the candy machine has to be added as one of the creators in order for it to show up. I'm guessing that was all of our issues.
> 
> Edit: Or any other wallet that is a creator. I'm guessing most of you are using a different wallet for the candy machine vs the storefront.

Hi, I followed ur instructions to add creator of my candy machine and creator in the metadata of nfts in the store. But still nothing shows up.
![image](https://user-images.githubusercontent.com/14337160/136499388-09f0b238-667d-4f50-bd5b-4c751ac8fd64.png)
![image](https://user-images.githubusercontent.com/14337160/136499446-8496f87b-ac3a-4628-a40b-4b4c6604ea23.png)
Do u know what is wrong here?

**Comment by SudarminThen** [comment]
I don't think this is a solution, it's only a workaround, what if someone else bought NFT from the candy machine that you don't know the address, and then his NFT wouldn't show up, and you don't know the candy machine id as well.
It would not be feasible to let users report all the candy machine id from their NFTs to us, so we can put it in as the creator
